presto-spi/src/main/java/com/facebook/presto/spi/function/WindowIndex.java

```diff
@@ -81,6 +81,14 @@
     Block getSingleValueBlock(int channel, int position);
 
     /**
+     * Gets an object value.
+     * @param channel argument number
+     * @param position row within the partition, starting at zero
+     * @return
+     */
+    Block getObject(int channel, int position);
```

Comment:
This should return type `Object` to match the `Pages.getObject`.  In the long run the plan is to support any object type from these method.